workflow/rules/common.smk

```diff
@@ -57,18 +57,33 @@ def get_multiqc_input(wildcards):
         reads = [ "1", "2" ]
         if is_single_end(sample, unit):
             reads = [ "0" ]
+            multiqc_input.extend(expand (["logs/cutadapt/{sample}-{unit}.se.log"],sample = sample,
+            unit = unit))
+        else:
+            multiqc_input.extend(expand (["logs/cutadapt/{sample}-{unit}.pe.log"],sample = sample,
```

Comment:
Very good place to add these files! I'd break up each extend statement a bit further, putting `sample = sample,` on a separate line and indenting it one level alongside `unit = unit`. An alternative (easy way) would be to run the python code formatter `black` on the whole project once in a while:

```
conda create -n black black
conda activate black
cd /path/to/folder/containing/repo
black chipseq/
```
It creates consistent formatting and all you have to do is `git add --update` and `git commit -m "code formatting with black"` to have it checked in.